src/Test.UnitTests.BinaryParsers/PEBinary/PEBinaryTests.cs

```diff
@@ -32,27 +32,40 @@ internal static string GetTestDirectory(string relativeDirectory)
             return Path.Combine(dirPath, relativeDirectory);
         }
 
-        [Fact]
-        public void PEBinary_PdbAvailable()
+        [Theory]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_default.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_4096.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_8192.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_16384.exe")]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_32768.exe")]
+        [InlineData("Native_x64_VS2022_PDBPageSize_8192.exe")]
+        [InlineData("Native_x64_VS2013_Default.dll", true)]
```

Comment:
This type of test adds extra latency to test runtime. We can get the same effect without the latency by adding a foreach loop over a list containing the same strings within the test.

Reply:
Oh I just discovered this syntax in the existing tests,
looks cool, did not know it is slow, let me change it to a loop.